src/main/java/core/basesyntax/FindNumbers.java

```diff
@@ -11,6 +13,19 @@
      * числа 92, 18, 26 и 0.</p>
      */
     public int[] getAllNumbers(String text) {
-        return null;
+
+        String[] numbersString = text.replaceAll("[^-?[1-9]\\d*|0]", ".")
```

Comment:
the regex is too complex
we need to replace everything that is not a `-` or a digit